protos/squareup/cash/pranadb/service/v1/service.proto

```diff
@@ -30,11 +34,34 @@ message Column {
 }
 
 // Issue a query.
-message ExecuteSQLStatementRequest {
+message ExecuteStatementRequest {
   string schema = 1;
   string statement = 2;
-  // Size of each page of results returned when paginating.
-  int32 page_size = 3;
+  int32 batch_size = 3;
+}
+
+message ExecutePreparedStatementRequest {
+  string schema = 1;
+  string statement_or_name = 2;
```

Comment:
Why `statement_or_name`?
Is the idea that in future you could pass the name of some stored procedure rather than a query string?
If so, would it make sense to use a `oneof` and have mutually-exclusive `statement`, and `name` fields when that feature is added?

Reply:
As an aside, do `ExecuteStatementRequest` and `ExecutePreparedStatementRequest` need to be separate? Wouldn't `ExecutePreparedStatementRequest` with an empty list in `ps_args` be equivalent to `ExecuteStatementRequest`?

Reply:
I'll change it to a oneof